prog/weaver/main.go

```diff
@@ -473,6 +470,20 @@ func canonicalName(f *mflag.Flag) string {
 	return ""
 }
 
+func checkpointFlags(router *weave.NetworkRouter) map[string]string {
+	flags := map[string]string{}
+	status := weave.NewNetworkRouterStatus(router)
+	for _, conn := range status.Connections {
+		if connectionName, ok := conn.Attrs["name"].(string); ok {
+			if _, encrypted := conn.Attrs["encrypted"]; encrypted {
+				connectionName = connectionName + " encrypted"
+			}
+			flags[connectionName] = "network"
```

Comment:
Is this expected to have `connectionName` as the key instead of as the value?
I may be missing something, but it looks like key and value have been swapped compared to: 
- `flags["network"] = network` which we had before, and 
- related logic [here](https://github.com/weaveworks/go-checkpoint/blob/a2ed17a2dd36296191268351ec9e16acbf2229f9/checkpoint.go#L148-L158).